pystac/summaries.py

```diff
@@ -84,21 +91,25 @@ def __repr__(self) -> str:
         return self.to_dict().__repr__()
 
 
-FIELDS_JSON_URL = (
-    "https://cdn.jsdelivr.net/npm/@radiantearth/stac-fields/fields-normalized.json"
-)
-
-
 @lru_cache(maxsize=None)
-def _get_fields_json(url: str) -> Dict[str, Any]:
+def _get_fields_json(url: Optional[str]) -> Dict[str, Any]:
+    if url is None:
+        # Every time pystac is released this file gets pulled from
+        # https://cdn.jsdelivr.net/npm/@radiantearth/stac-fields/fields-normalized.json
+        jsonfields: Dict[str, Any] = json.loads(
+            importlib_resources_files("pystac.static")
+            .joinpath("fields-normalized.json")
+            .read_text()
+        )
+        return jsonfields
     return pystac.StacIO.default().read_json(url)
 
 
 class SummaryStrategy(Enum):
     ARRAY = "v"
     RANGE = "r"
     SCHEMA = "s"
-    DONT_SUMMARIZE = False
+    DO_NOT_SUMMARIZE = False
```

Comment:
On second thought, can we pull this out of this PR? _Technically_ this is a breaking change so should go in a v2 milestone.

Reply:
yes I can take this out of this PR